static/js/teachers.ts

```diff
@@ -133,7 +133,7 @@ export function invite_student(class_id: string) {
       }).done(function() {
           location.reload();
       }).fail(function(err) {
-          return modal.alert(err.responseText);
+          return modal.alert(err.responseText, 3000, true);
```

Comment:
Is the 3000 here a color? Maybe we can make it into a constant, or add a comment because like this it is hard to know what it means.

Reply:
My bad! The 3000 is the timeout in ms. Will add an inline comment to explain this. However, we use the same structure in a lot of places when calling modals. Will take a look if we can improve the readability.

Reply:
Ah so then I don't know why this change changes color? It is because of the true (Note that I did not read the rest of the code, just this snippet!)

Reply:
Yes it's a boolean indicating an error or a success message. When true the message is shown in red, otherwise in green.

Reply:
Ah got it!

Reply:
Does ts allow for named parameters?

 `return modal.alert(err.responseText, timeout=3000, error=true);` would be a lot more understandable?

Reply:
Ah it seems it does not really understand, and I do agree it is overkill to document it at all calls, so let's leave it the way it is.